apps/website/app/api/embeddings/openai/small/route.ts

```diff
@@ -0,0 +1,86 @@
+import { NextRequest, NextResponse } from "next/server";
+import OpenAI from "openai";
+import cors from "~/utils/llm/cors";
+
+const apiKey = process.env.OPENAI_API_KEY;
+
+if (!apiKey) {
```

Comment:
We could probably throw an error here because it won't work at all without this.

This will make further code more type safe and unnecessary  like the `?` and `!` checks.